src/components/BodyPage.vue

```diff
@@ -0,0 +1,83 @@
+<script>
+    export default {
+        name: 'BodyPage',
+        data() {
+            return {
+                selected: 'Test'
+            }
+        }
+    }
+</script>
+    
+<template>
+    <section class="c-section__filters">
```

Comment:
This does not have to be **c-section__filters**.
It's basically a new component which you could call **c-filters**